### PR TITLE
contains device history page, CRUD operations from Admin, Checkin and…

### PR DESCRIPTION
**Pull request by dhanshrip-cuelogic**
… Checkout functionality bye Employee

### CODE REVIEW

**Comment by dhanshrip-cuelogic** [comment]
@VikaspCuelogic @swapnils-cuelogic

File: DeviceInventory/File Manager/DatabaseManager.swift [hunk 1]
```diff
@@ -16,32 +16,164 @@ class DatabaseManager {
...
+//    var checkinDetails = [IssuedDevices]()
+    
     private (set) var fetchedData : [DeviceDetails]? {
         didSet {
              NotificationCenter.default.post(name: Notification.Name(rawValue: "loadedPost"), object: nil)
         }
     }
+    private (set) var issuedDeviceData : [IssuedDevices]? {
+        didSet {
+             NotificationCenter.default.post(name: Notification.Name(rawValue: "gotIssuedData"), object: nil)
```
Comment (VikaspCuelogic) [inline DeviceInventory/File Manager/DatabaseManager.swift:30]: use closure or protocol to update data to the presenter

File: DeviceInventory/Model/IssuedDevices.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//  IssuedDevices.swift
+//  DeviceInventory
+//
+//  Created by Dhanshri Pawar on 06/05/20.
+//  Copyright © 2020 Dhanshri Pawar. All rights reserved.
+//
+
+import Foundation
+
+struct IssuedDevices : Codable {
```
Comment (VikaspCuelogic) [inline DeviceInventory/Model/IssuedDevices.swift:11]: follow naming conventions for variable declaration

**Review by VikaspCuelogic — CHANGES_REQUESTED** [review]
follow naming conventions while declaring methods and variables
avoid use of notification until it is required

**Pull request merged**